packages/components/toggle-button/src/toggle-button.ts

```diff
@@ -13,8 +13,10 @@ declare global {
   }
 }
 
-export type ToggleButtonFill = 'ghost' | 'outline';
-export type ToggleButtonSize = 'md' | 'lg';
+export type ToggleButtonFill = 'ghost' | 'outline' | 'solid';
+export type ToggleButtonSize = 'sm' | 'md' | 'lg';
+export type ToggleButtonShape = 'pill' | 'square';
```

Comment:
Do we need to define this ourselves? Why not use `ButtonShape`, or at least do `export type ToggleButtonShape = ButtonShape;`?